internal/db/schema/migrations/postgres/14/01_event.up.sql

```diff
@@ -0,0 +1,250 @@
+
+create table event_type_enabled (
+    config_id wt_public_id
+        constraint event_config_fkey
+            references event_config(public_id)
+            on delete cascade
+            on update cascade,
+    event_type text not null
+        constraint event_type_enm_fkey
+            references event_type_enm (name)
+            on delete restrict
+            on update cascade,
+    primary key (config_id, event_type)
+);
+comment on table event_type_enabled is
+'event_type_enabled is a table where each row represents that eventing has '
+'been enabled for the specified event type in an event configuration';
+
+
+create table event_sink(
+    public_id wt_public_id primary key,
+    config_id wt_public_id 
+        constraint event_config_fkey
+            references event_config(public_id)
+            on delete cascade
+            on update cascade
+);
+comment on table event_sink is 
+'event_sink is a table where each row represents a configured event sink';
+
+create table event_file_sink(
+    public_id wt_public_id primary key,
+    sink_id wt_public_id not null
+        constraint sink_id_fkey
+            references event_sink(public_id)
+            on delete cascade
+            on update cascade,
+    event_type text not null
+        constraint event_type_enm_fkey
+            references event_type_enm(name)
+            on delete restrict
+            on update cascade,
+    format_type text not null
+        constraint event_format_type_enm_fkey
+            references event_format_type_enm(name)
+            on delete restrict
+            on update cascade,
+    allow_filter wt_bexprfilter,
+    deny_filter wt_bexprfilter,
+    path text not null 
+        constraint path_not_empty
+            check (
+                length(trim(path)) > 0
+            ),
+    filename text not null
+        constraint filename_not_empty
+            check (
+                length(trim(filename)) > 0
+            ),
+        constraint path_filename_uq
+            unique(path, filename), -- ensure each sink is writing to a unique file
+    rotate_bytes int
+        constraint rotate_bytes_null_or_greater_than_zero
+            check(
+                rotate_bytes is null
+                    or
+                rotate_bytes > 0
+            ),
+    rotate_duration interval
+        constraint rotate_duration_null_or_greater_than_zero
+            check(
+                rotate_duration is null
+                    or
+                rotate_duration > '0'::interval
+            ),
+    rotate_max_files int
+        constraint rotate_max_files_null_or_greater_than_zero
+            check(
+                rotate_max_files is null
+                    or
+                rotate_max_files > 0
+            )
+);
+comment on table event_file_sink is 
+'event_file_sink is a table where each entry represents a configured event file '
+'sink';
+
+create table event_stderr_sink(
+    public_id wt_public_id primary key,
```

Comment:
Is this table missing one or more unique constraints since the primary key is a surrogate key?